pkg/utils/common/controller.go

```diff
@@ -229,6 +229,20 @@ func (s *SuiteController) DeleteConfigMap(name, namespace string) error {
 	return s.KubeInterface().CoreV1().ConfigMaps(namespace).Delete(context.TODO(), name, metav1.DeleteOptions{})
 }
 
+//CheckIfNamespaceExists - find if a namespace exists
+func (h *SuiteController) CheckIfNamespaceExists(name string) bool {
+
```

Comment:
You don't need an empty line at the beginning.

Reply:
fixed